include/pinocchio/spatial/inertia.hpp

```diff
@@ -424,6 +428,37 @@ namespace pinocchio
       return *this;
     }
 
+    InertiaTpl __minus__(const InertiaTpl & Yb) const
+    {
+      /* Y_{a-b} = ( m_a-m_b,
+       *             (m_a*c_a - m_b*c_b ) / (m_a - m_b),
+       *             I_a - I_b + (m_a*m_b)/(m_a-m_b) * AB_x * AB_x )
+       */
+
+      const Scalar eps = ::Eigen::NumTraits<Scalar>::epsilon();
+
+      const Scalar & mab = mass()-Yb.mass();
+      const Scalar mab_inv = (mass()-Yb.mass() >= 0) ? Scalar(1)/math::max((Scalar)(mass()-Yb.mass()),eps) : -Scalar(1)/math::max((Scalar)(Yb.mass()-mass()),eps);
```

Comment:
I don't think that the case `mab < 0` has a physical meaning.

Reply:
This depends on how we want to design this class. At the moment, it is unclear to me. See my comment at https://github.com/stack-of-tasks/pinocchio/pull/2204#discussion_r1552596106.